threeML/plugin_prototype.py

```diff
@@ -125,3 +125,14 @@ def inner_fit(self):
         logLike value.
         """
         pass
+
+    @abc.abstractmethod
```

Comment:
Please do not make this an abstract method. Instead, provide here an implementation that issue a warning like "get_number_of_data_points not implemented, values for statistical measurements such as AIC or BIC are unreliable", and then returns 1. 

This way you can remove the placeholders in all plugins, and we will remember  that we need to fix this eventually.